README.md

```diff
@@ -1,73 +1,29 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+#
+# mobile Version
```

Comment:
- [ Optional ] We apologize for not pointing it out earlier, but all sections in the README [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse are mandatory to have in your README because it helps make your README look professional. So, please add the description of the Portfolio project here.